LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -1193,9 +1193,16 @@ BriefSummary="Massively reduces weapon noise and the soldier's impact on mission
 TinySummary="Reduces mission infiltration time"
 LootTooltip="This weapon upgrade, when installed, will reduce the time the soldier requires to infiltrate during a mission. It also reduces weapon noise during combat."
 
+[CritUpgrade_Bsc X2WeaponUpgradeTemplate]
```

Comment:
Please wrap changed lines with the following block as described in [the ikiw](https://github.com/long-war-2/lwotc/wiki/Contributing#localization-translating-text-in-the-game):

```
; LWOTC Needs Translation
...
; End Translation
```